include/cuco/detail/probing_scheme_base.cuh

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cstdint>
+
+namespace cuco {
+namespace experimental {
+namespace detail {
+/**
```

Comment:
```suggestion

/**
```

Reply:
I'm accepting all those changes of adding a new line right after the namespace but there are definitely places we missed. Any particular reason for having this new line? If so, we may want to enforce it via a CI script.

Reply:
Just for consistency reasons I guess. I'm fine with whatever CI is fine with 😅